Release 1.10.0 (develop) (#397)
SwedbankPay/developer.swedbankpay.com: resources/release-notes.md

## May 2020

### Version 1.10.0

> * Added a new section [Gift Cards][gift-cards] for Payment API documentation.

Reviewer: ```suggestion
* Added a new [Gift Cards][gift-cards] section with descriptions of our Gift Cards API.
```
